### DISCUSSION TITLE
Trying to add a new parameter

### DISCUSSION THREAD

**Discussion by wass-cloud** [pybamm-team/PyBaMM#3210] [Q&A]
I'm trying to add the parameter ['Typical plated lithium concentration [mol.m-3]'] to my code due to an error I'm getting where it states that this parameter has no value, however when I do 

pybamm.ParameterValues.__setitem__(param, key = 'Typical plated lithium concentration [mol.m-3]', value = 534 / (6.94e-03))

which should use the update() method to create a new item in the dictionary, instead i am getting the following error:

"Cannot update parameter 'Typical plated lithium concentration [mol.m-3]' as it does not have a default value." 

From my experience with python this should not occur, it would just add the new key and value to the dictionary, there would be no need for a default value.

**Comment by valentinsulzer** [comment]
Use `parameter_values.update({"Typical plated lithium concentration [mol.m-3]": 534 / (6.94e-03)}, check_already_exists=False)`
